__tests__/app.test.js

```diff
@@ -167,7 +151,29 @@ describe("PATCH /api/articles/:article_id", () => {
       .then((response) => {
         expect(response.body.status).toBe(400);
         expect(response.body.msg).toBe("bad request - invalid request body");
-
+      });
+  });
+});
+describe("GET /api/users", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { users } = response.body;
+        expect(Array.isArray(users)).toBe(true);
+      });
+  });
+  it("should return an array of user objects with the correct keys", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { users } = response.body;
+        expect(users.length).not.toBe(0);
+        users.forEach((user) => {
+          expect(user.hasOwnProperty("username")).toBe(true);
+        });
       });
   });
 });
```

Comment:
Consider using the 'objectContaining' matcher here to ensure that username is a key and that the value is a string..